include/statespace/StateSpace.hpp

```diff
@@ -160,6 +160,12 @@ class StateSpace::State {
     /// \return Output state vector
     virtual Eigen::VectorXd vector() const = 0;
 
+    /// Converts state value given vector representation
+    /// Throws and exception if state vector is of wrong format
```

Comment:
if the state vector size is incorrect; allowed vector size depends on derived class